projects/rero/ng-core/src/lib/api/api.service.ts

```diff
@@ -45,19 +43,32 @@ export class ApiService {
   }
 
   /**
-   * Returns invenio API Endpoint corresponding to type.
+   * Returns Invenio API Endpoint corresponding to type.
    *
    * @param type Type of the resource.
    * @param absolute If absolute or relative url must be returned.
    * @return Endpoint as string.
    */
   getEndpointByType(type: string, absolute: boolean = false): string {
     let endpoint = this.endpointPrefix + '/' + type;
-
     if (absolute === true) {
       endpoint = this.baseUrl + endpoint;
     }
+    return endpoint;
+  }
 
+  /**
+   * Returns Invenio-RERO-base API export endpoint corresponding to type.
+   *
+   * @param type Type of the resource.
+   * @param absolute If absolute or relative url must be returned.
+   * @return Endpoint as string.
+   */
+  getExportEndpointByType(type: string, absolute: boolean = false): string {
+    let endpoint = this.endpointPrefix + '/export/' + type;
```

Comment:
there should be a configuration to allow customization of this entry

Reply:
This implies that this export endpoint URL should be customizable but at this time, it isn't. The new export module from 'invenio-rero-base' creates endpoint based on resource "list_route" prepend with static "/export" string.

Reply:
You an use back quotes instead of string concatenation.